joss_paper/paper.md

```diff
@@ -0,0 +1,260 @@
+  - name: Deepankar Sharma
+  - name: Erik M. Bray
+  - name: Jai Ram Rideout
+    orcid: 0000-0003-2587-1454
+    affiliation: "Dogfox Software LLC, Flagstaff, AZ 86001, USA"
+  - name: Serge Zahniy
+  - name: Tomas Meszaros
+  - name: Abhigyan Bose
+  - name: Andre Chicrala
+  - name: Ankit
+  - name: Chloé Guennou
+  - name: Daniel D'Avella
+  - name: Daniel Williams
+    affiliation: "SUPA, University of Glasgow, Glasgow G12 8QQ, United Kingdom"
+    orcid: 0000-0003-3772-198X
+  - name: Jordan Ballew
+  - name: Nick Murphy
+  - name: Priyank Lodha
+  - name: Thomas Robitaille
+  - name: Yash Krishan
+  - name: Andrew Hill
+  - name: Arthur Eigenbrot
+    orcid: 0000-0003-0810-4368
+    affiliation: "National Solar Observatory" 
+  - name: Benjamin Mampaey
+    orcid: 0000-0001-6541-4966
+    affiliation: Royal Observatory of Belgium
+  - name: Bernhard M. Wiedemann
+  - name: Carlos Molina
+    orcid: 0000-0003-0300-4106
+  - name: Duygu Keşkek
+  - name: Ishtyaq Habib
+  - name: Joseph Letts
+    orcid: 0000-0001-9900-739X
+  - name: Juanjo Bazán
+    orcid: 0000-0001-7699-3983
+    affiliation: CIEMAT, Astroparticle physics, Madrid, Spain.
+  - name: Quinn Arbolante
+    affiliation: Lockheed Martin Solar and Astrophysics Laboratory
+    orcid: 0000-0003-0260-453X
+  - name: Reid Gomillion
+  - name: Yash Kothari
+  - name: Yash Sharma
+  - name: Abigail L. Stevens
+    orcid: 0000-0002-5041-3079
+    affiliation:
+      - 'Department of Physics & Astronomy, Michigan State University, 567 Wilson Road, East Lansing, MI 48824, USA'
+      - 'Department of Astronomy, University of Michigan, 1085 South University Avenue, Ann Arbor, MI 48109, USA'
+  - name: Adrian Price-Whelan
+    orcid: 0000-0003-0872-7098
+    affiliation: Center for Computational Astrophysics, Flatiron Institute, 162 Fifth Ave, New York, NY 10010, USA
+  - name: Ambar Mehrotra
+  - name: Arseniy Kustov
+  - name: Brandon Stone
+  - name: Trung Kien Dang
+    orcid: 0000-0001-7562-6495
+    affiliation: Saw Swee Hock School of Public Health, National University Health System, National University of Singapore, Singapore
+  - name: Emmanuel Arias
+  - name: Fionnlagh Mackenzie Dover
+    orcid: 0000-0002-1984-7303
+    affiliation: "SP2RC, School of Mathematics and Statistics, The University of Sheffield, UK"
+  - name: Freek Verstringe
+    affiliation: "Royal Observatory of Belgium"
+  - name: Gulshan Kumar
+    orcid: 0000-0001-8523-7223
+    affiliation: International Institute of Information Technology, Hyderabad
+  - name: Harsh Mathur
+    orcid: 0000-0001-5253-4213
+    affiliation: Indian Institute of Astrophysics, Bangalore
+  - name: Igor Babuschkin
+  - name: Jaylen Wimbish
+  - name: Juan Camilo Buitrago-Casas
+  - name: Kalpesh Krishna
+  - name: Kaustubh Hiware
+    orcid: 0000-0003-3301-1016
+    affiliation: "Indian Institute of Technology, Kharagpur"
+  - name: Manas Mangaonkar
+  - name: Matthew Mendero
+  - name: Mickaël Schoentgen
```

Comment:
```suggestion
  - name: Mickaël Schoentgen
    orcid: 0000-0002-0106-4810
```

Reply:
FTR I have no affiliation, and do not want to put my current employer as it is not related. Is it OK?

Reply:
That's no problem :smile: